plenum/server/replica.py

```diff
@@ -761,15 +761,13 @@ def create3PCBatch(self, ledger_id):
             if last_ordered_ts:
                 self.last_accepted_pre_prepare_time = last_ordered_ts
 
-        resp = self.consume_req_queue_for_pre_prepare(ledger_id, self.viewNo,
+        validReqs, inValidReqs, rejects, tm = self.consume_req_queue_for_pre_prepare(ledger_id, self.viewNo,
                                                       pp_seq_no)
-        if not resp:
+        if not (validReqs or inValidReqs):
```

Comment:
Please add a unit test for this (see `test/replica/test_api`)

Reply:
Are we need to check that create3PCBatch() doesn't return value or log message too (and change logLevel )?